types/filter.go

```diff
@@ -42,7 +42,7 @@ func (f *EventFilter) Validate() error {
 		return errors.New("filter must have one or more statements")
 	}
 
-	if err := eval.ValidateStatements(f.Statements, false); err != nil {
+	if err := js.ParseExpressions(f.Statements); err != nil {
```

Comment:
"Statements" here should probably be "Expressions" (just a quick note in case you miss that one)